var/spack/repos/builtin/packages/seacas/package.py

```diff
@@ -133,6 +136,7 @@ class Seacas(CMakePackage):
 
     # ###################### Dependencies ##########################
     depends_on("cmake@3.17:", type="build")
+    depends_on("cmake@3.22:", when="@2023-10-24", type="build")
```

Comment:
```suggestion
    depends_on("cmake@3.22:", when="@2023-10-24:", type="build")
    depends_on("cmake@3.17:", when="@:2023-05-30", type="build")
```
These should probably be ranges, and dependencies are usually ordered so newer versions are before older ones.

Reply:
done i think.  Is `master` considered after `2023-10-24`?